.travis.yml

```diff
@@ -5,4 +5,4 @@ node_js:
 env:
 - TEST_DIR=.
 - TEST_DIR=scripts/babel-relay-plugin
-script: cd $TEST_DIR && npm install && npm test
+script: cd $TEST_DIR && npm install && npm run lint && npm test
```

Comment:
I would probably do this in 2 totally different PRs.

1. Make lint work
2. Add it to travis